instructor/templates/instructor/generic_form.html

```diff
@@ -0,0 +1,14 @@
+{% extends "instructor/assignment_sidebar.html" %}
+
+{% block right_side %}
+    Assignment: {{ assignment.name }} - {{ title }}
+    {% if message %}
+        {{ messsage }}
+    {% endif %}
+    <div>
+        <form method="post">{% csrf_token %}
+            {{ form.as_p }}
+            <input type="submit" class="scb_s_assignment_setup_save_button scb_s_navigation_button" value="Submit Questions"/>
+        </form>
+    </div>
+{% endblock %}
```

Comment:
new line, also the file name could use some work.  Maybe `assignment_generic_sub_form.html` or similar